packages/core/src/components/graphs/bar.ts

```diff
@@ -10,7 +10,7 @@ export class Bar extends Component {
 			return options.bars.width;
 		}
 
-		const numberOfDatapoints = this.model.getDisplayData().length;
```

Comment:
sorry...!

I actually made a mistake here.

We should keep the override rather than just updating the `bar.ts` code. I think once we revert that piece we're good to merge.

Reply:
hi @theiliad, I have reverted last commit